docs/guides/cloud/introduction.mdx

```diff
@@ -4,157 +4,152 @@ sidebar_position: 10
-no setup required! You can prioritize recently failed specs with
-[Spec Prioritization](/guides/cloud/smart-orchestration/spec-prioritization) to
-surface problems earlier, and cancel whole test runs on failure with
-[Auto Cancellation](/guides/cloud/smart-orchestration/run-cancellation) to save
-on resource usage. You can also
-[cancel in-progress runs](/guides/cloud/recorded-runs#Run-cancellation) manually
-from Cypress Cloud if you need to.
+[balances test loads](/guides/cloud/smart-orchestration/load-balancing#Balance-strategy) - no setup required!
 
 <DocsImage
   src="/img/dashboard/introduction/orchestration-diagram.png"
   alt="Diagram comparing serial and parallel test configurations"
 />
 
-### Integrate with source control providers
-
-Ensure rock-solid reliability by keeping failing code out of your
-[GitHub](https://github.com), [GitLab](https://gitlab.com) and
-[Bitbucket](https://bitbucket.org) repositories, with
-[status checks](/guides/cloud/integrations/source-control/github#Status-checks)
-that block commits from being merged until your Cypress tests are green. Surface
-test results directly in your PRs with
-[pull request comments](/guides/cloud/integrations/source-control/github#Pull-request-comments)
-that include test run statistics, specific failure details, and deep links to
-results in Cypress Cloud for fast debugging. Users with
-[Business or Enterprise plans](https://www.cypress.io/pricing/) can integrate
-securely with
-[Self-managed GitLab](https://docs.gitlab.com/ee/subscriptions/self_managed/)
-instances too.
+Cypress Cloud's
+[Spec Prioritization](/guides/cloud/smart-orchestration/spec-prioritization)
+lets you prioritize recently failed specs so they run faster on their next run.
+The [Auto Cancellation](/guides/cloud/smart-orchestration/run-cancellation)
+feature will cancel a whole test run on failure, saving time and resource usage.
+You can also [cancel in-progress runs](/guides/cloud/recorded-runs#Run-cancellation)
+manually from Cypress Cloud if you need to.
+
+### Integrate with your favorite tools
+
+#### Source Control providers
+
+Cypress Cloud integrates with [GitHub](/guides/cloud/integrations/source-control/github),
+[GitLab](/guides/cloud/integrations/source-control/gitlab), and
+[Bitbucket](/guides/cloud/integrations/source-control/bitbucket) to provide rock-solid
+reliability by keeping failing code out of your main branch. Use status checks
+to block commits from being merged until your Cypress tests are green, and
+surface test results directly in your PRs with pull request comments that
+include test run statistics, specific failure details, and deep links to results
+in Cypress Cloud for fast debugging.
 
 <DocsImage
   src="/img/dashboard/github-integration/status-checks-per-spec.jpg"
   alt="Status checks per spec"
 />
 
-### Collaborate and organize
+[Business and Enterprise plans](https://www.cypress.io/pricing/) include secure
```

Comment:
This line kind of gets lost in the flow. It should be fine to move it up as a separate paragraph, but above the screenshot.